src/components/Error/ErrorDialog.tsx

```diff
@@ -51,6 +53,10 @@ const ExpandoContent = styled(ThemedText.Code)`
   margin: 0.5em 0;
 `
 
+const StyledXButton = styled(X)`
```

Comment:
could you possibly reuse the StyledXButton component / export it instead of declaring twice in 2 separate files? although not sure what widget codebase convention is, so this is more of a suggestion / nit

Reply:
thanks, i'll dedup these